server/index.js

```diff
@@ -18,6 +22,15 @@ app.use(bodyParser.urlencoded({ extended: false }));
 
 app.use('/api-docs', swaggerUi.serve, swaggerUi.setup(swaggerConfig));
 
+// add worker if on production environment
+if (process.env.NODE_ENV === 'production') {
+  setInterval(async () => {
+    worker();
```

Comment:
The `worker` should be started after the application has started listening i.e, in the call back of `app.listen`. Doing this would help ensure that the worker is not started if the app itself is not started.